lib/config.js

```diff
@@ -14,7 +14,7 @@ const RPC_URL = rpcUrls[Math.floor(Math.random() * rpcUrls.length)]
 console.log(`Selected JSON-RPC endpoint ${RPC_URL}`)
 
 const rpcHeaders = {}
-if (RPC_URL.includes('glif')) {
+if (RPC_URL.includes('glif') && GLIF_TOKEN) {
```

Comment:
```suggestion
if (RPC_URL.includes('glif')) {
```

The glif token is _always_ required

Reply:
Unfortunately, our tests don't provide GLIF_TOKEN in the env vars. I had a different implementation initially, it worked when I run `node bin/dry-run.js` but the tests started to fail.

Reply:
be9f36f lgtm!

> With my change in place, instead of sending invalid authorisation header, we don't send any. I think the outcome should be the same.

I see! I like sending an invalid auth header better, since in the code we're baking in the assumption that it will be provided. Otherwise, as a reader with little context, this line doesn't make sense to me.